jujugui/static/gui/src/app/app.js

```diff
@@ -491,7 +491,7 @@ YUI.add('juju-gui', function(Y) {
       let password = null;
       let macaroons = null;
       let credentials = this.user.controller;
-      if (credentials.areAvailable) {
+      if (credentials && credentials.areAvailable) {
```

Comment:
Now I also see the areAvailable error in the api object. So there it's ok to check if user is defined I guess. But in the app initializer we have `this.user = this.get('user') || new window.jujugui.User();` so I thing we always have the user object available.